lib/FieldType/Mapper/RichTextFormMapper.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use eZ\Publish\Core\FieldType\RichText\Type;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+class RichTextFormMapper implements FieldTypeFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add(
+                'numRows',
+                'integer',
+                [
+                    'empty_data' => 10,
+                    'required' => true,
+                    'property_path' => 'fieldSettings[numRows]',
+                    'label' => 'field_definition.ezrichtext.num_rows',
+                ]
+            )
+            ->add(
+                'tagPreset',
```

Comment:
I agree it's more a PM decision, but in a first step, I would not add this settings because given the new behaviour of the RichText editor, the setting would most likely to be a bit different.